.gitignore

```diff
@@ -6,14 +6,13 @@ public/
 newman/
 .idea/
 .venv/
-.vscode/
```

Comment:
Is this git being weird? probably want one of these still in there

Reply:
No, we want to allow .vscode so that we can keep the launch.json for local dev.